backend/src/main/java/hu/flowacademy/meteo/dto/MeasurmentDto.java

```diff
@@ -16,25 +16,13 @@
 @Builder
 public class MeasurmentDto {
 
-    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
+    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+2")
```

Comment:
Itt nem foglalkoznek time zone-nal, backenden felesleges, a frontend kapjon egy standard mondjuk iso formatumot, es a kliens oldalon a bongeszo locale-janak megfeleloen bekonvertalja es megjeleniti ahogy akarja.
Ha emlkeeyetem nem csal akkor ha nem adsz meg formazast akkor Z vegu timestamp stringet kuld ki (ez jeloli az UTC-t). Erdemes majd ha lesz kis idotok letesztelni a csv beolvasast hogy valoban valid date-eket ment-e el.

Reply:
Valahol feljebb én is írtam, ez minden DTO-ban benne van, és  a backenden antipattern.